pkg/runtime/validate.go

```diff
@@ -102,6 +103,11 @@ func (e *Executor) ValidateInputs() error {
 	}
 	zap.S().Debugf("using policy path %v", e.policyPath)
 
+	if len(e.severity) > 0 && !utils.ValidateSeverityInput(e.severity) {
+		return errSeverityNotSupported
```

Comment:
should we error out when incorrect severity is passed by user?
I think we should only log regarding it and proceed with the scan.

Reply:
IMO, we should explicitly inform the user about their human error. A user might enter a invalid value "hihg" expecting only HIGH severity violations to be reported. But if assume the defaults and proceed with scan in this case, they'll get reported the violations of all the severity levels, opposed to the user's expectation. This erroring out way will help the user make robust decisions while creating CI / CD pipelines as well.

WDYT @williepaul @kanchwala-yusuf @jlk @acc-jon @amirbenv

Reply:
IMO, the scan command (or any other command) should error out only when an option value is critical for the scan command to proceed eg: the file/directory passed for scan.

Reply:
I think if a user specifies a severity, and inputs an error, we should error out and tell them they inputted an unsupported severity. Otherwise, the scan will not run as they expect, and they could either not notice, or notice and think terrascan has a bug.

Reply:
I think we should treat it similar to other "picklist" options like -i: there are specific valid values for the option, and invalid options should trigger a usage error.